test/tests/kernels/ode/tests

```diff
@@ -22,4 +22,13 @@
     max_parallel = 2
     max_threads = 2
   [../]
+
+  [./test_coupled_ode_t]
+    prereq = 'test_sys_impl'
+    type = 'Exodiff'
+    input = 'ode_expl_test.i'
+    exodiff = 'ode_expl_test_out.e'
+    max_parallel = 2
+    max_threads = 2
+  [../]
```

Comment:
Copy and paste problems:

1. you do not need the `prereq`
2. wrong input file, should be `coupled_ode_td.i`.
3. wrong output file